src/main/studio/kdb/Config.java

```diff
@@ -439,11 +439,20 @@ public boolean isShowServerComboBox() {
         return Boolean.parseBoolean(p.getProperty("showServerComboBox","true"));
     }
 
+    public boolean isShowConsoleViewComboBox() {
```

Comment:
Why do you include "ComboBox" in the name? It looks more like it is a button that is enabled by the setting.

Reply:
it was just to be consistent with the showServer part even though i agree the name doesn't make a ton of sense as it's actually a "checkbox". Maybe it used to be a combobox and function name didn't get updated....

Reply:
The server one is actually a combo box (colloquially it's called a dropdown list). So in that case the name make sense.
You need to name the new variable to reflect what it affects. From my reading of the code it seems it's the console view itself. So just calling it "showConsoleView" would be better.

Reply:
I get what you're saying - updated the PR